sopel/modules/clock.py

```diff
@@ -250,11 +258,12 @@ def update_channel(bot, trigger):
 
 
 @module.commands('getchanneltz', 'getctz')
-@module.example('.getctz [channel]')
+@module.example('.getctz #sopel', user_help=True)
+@module.example('.getctz', user_help=True)
 def get_channel_tz(bot, trigger):
-    """
-    Gets the channel's preferred timezone; returns the current channel's
-    if no channel name is given.
+    """Gets the channel's preferred timezone.
+
+    It uses the current channel's if no channel name is given.
```

Comment:
```suggestion
    It uses the current channel if no channel name is given.
```

OR, for consistency with `setctf`:

```suggestion
    It returns the current channel's if no channel name is given.
```